.github/workflows/stm32-build.yaml

```diff
@@ -6,7 +6,15 @@
 
 name: STM32 Build
 
-on: [push, pull_request]
+on:
+  push:
+    paths:
+      - 'src/platforms/stm32/**'
+      - 'src/libAtomVM/**'
+  pull_request:
+    paths:
+      - 'src/platforms/stm32/**'
+      - 'src/libAtomVM/**'
 
```

Comment:
We probably want to also include:
- .github/workflows
- CMakeLists.txt

Reply:
Top level `CMakeLists.txt` should not be required here either...

Unless we want to include theme in these places as a precaution for future changes to the build system...